### PR TITLE
Allow ASCII fallback for Unicode characters in logs

### PR DESCRIPTION
**Pull request by dtardon**

### CODE REVIEW

File: src/libsystemd/sd-bus/sd-bus.c [hunk 1]
```diff
@@ -521,7 +521,7 @@ void bus_set_state(sd_bus *bus, enum bus_state state) {
         if (state == bus->state)
                 return;
 
-        log_debug("Bus %s: changing state %s → %s", strna(bus->description), table[bus->state], table[state]);
+        log_debug("Bus %s: changing state %s -> %s", strna(bus->description), table[bus->state], table[state]);
```
Comment (yuwata) [inline src/libsystemd/sd-bus/sd-bus.c]: Why not to use special_glyph() here?

File: src/libsystemd/sd-event/sd-event.c [hunk 2]
```diff
@@ -405,7 +405,7 @@ _public_ int sd_event_new(sd_event** ret) {
         e->epoll_fd = fd_move_above_stdio(e->epoll_fd);
 
         if (secure_getenv("SD_EVENT_PROFILE_DELAYS")) {
-                log_debug("Event loop profiling enabled. Logarithmic histogram of event loop iterations in the range 2^0 … 2^63 us will be logged every 5s.");
+                log_debug("Event loop profiling enabled. Logarithmic histogram of event loop iterations in the range 2^0 ... 2^63 us will be logged every 5s.");
```
Comment (yuwata) [inline src/libsystemd/sd-event/sd-event.c]: Also here.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: src/libsystemd/sd-bus/sd-bus.c (hunk 1 above)
Comment (dtardon) [reply src/libsystemd/sd-bus/sd-bus.c]: Hmm, why not indeed... `libsystemd` can use stuff from `basic`... Updated.

File: src/libsystemd/sd-event/sd-event.c (hunk 2 above)
Comment (dtardon) [reply src/libsystemd/sd-event/sd-event.c]: Fixed too.

**Pull request merged**